domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimDocument.java

```diff
@@ -7,7 +7,7 @@
 import java.time.LocalDateTime;
 
 @Getter
-public class ClaimDocument {
+public class ClaimDocument extends CollectionId {
```

Comment:
not specifically relevant to your PR but I don't like `CollectionId` name 😸, should this be something `CCDCollectionItem` or `CollectionItemWithId` - its not an ID, it's something with an ID.